lighthouse-core/report/html/renderer/performance-category-renderer.js

```diff
@@ -159,9 +177,11 @@ class PerformanceCategoryRenderer extends CategoryRenderer {
       const groupEl = this.renderAuditGroup(groups['diagnostics'], {expandable: false});
       diagnosticAudits.forEach((item, i) => groupEl.appendChild(this.renderAudit(item, i)));
       groupEl.open = true;
+      groupEl.classList.add('lh-audit-group--adorned', 'lh-audit-group--diagnostics');
```

Comment:
seems like the unadorned case is far more common, WDYT about it being the default to avoid this churn?

Reply:
Not sure but i'll give it a shot.